packages/calcite-components/src/demos/_assets/demo-theme.ts

```diff
@@ -15,10 +15,11 @@ export function getTokenValue(token: string): string {
     color$: "rgb(0, 191, 255)",
     hover$: "rgb(255, 105, 180)",
     pressed$: "rgb(44, 44, 44)",
+    selected$: "rgb(156, 89, 209)",
     shadow$:
       "rgb(255, 255, 255) 0px 0px 0px 4px, rgb(255, 105, 180) 0px 0px 0px 5px inset, rgb(0, 191, 255) 0px 0px 0px 9px",
-    "z-index$": "42",
-    "(size|space)$": "42px",
+    "(z-index)$": "42",
+    "(columns|gap|height|offset|radius|size|space|width)": "42px",
```

Comment:
fixme: Could this demo-theme code share the same code from `cssTokenValues`?

Reply:
where should it live? I was getting errors when I put a single file in /utils that it wasn't available to both storybook, puppeteer, and the html pages.

Reply:
@jcfranco @benelan do you know where we can put files used between all these areas?

Reply:
We could look into providing utils in the dev/demo build, but for now, we can proceed with the slight duplication.

Reply:
Created follow-up issue: https://github.com/Esri/calcite-design-system/issues/10107.